### ISSUE TITLE
Add basic kernel driver

### ISSUE THREAD

**Issue by JonathanBerkeley** [JonathanBerkeley/Quack#17]
Having a kernel driver would open the anti-cheat to new capabilities and a higher resilience to tampering. The first step towards this would be to create a basic kernel driver and have a method to load it that works in a generic way.

**Comment by moongazer07** [comment]
> Having a kernel driver would open the anti-cheat to new capabilities and a higher resilience to tampering. The first step towards this would be to create a basic kernel driver and have a method to load it that works in a generic way.

It might put some players off, since kernel mode anticheats can be pretty invasive.

**Comment by JonathanBerkeley** [comment]
> It might put some players off, since kernel mode anticheats can be pretty invasive.

That's true. My intention was an optional component, but this project is not intended for real world usage anyways.
